pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -1224,12 +1258,16 @@ func TestNodeOperators(t *testing.T) {
 				t.Error(err)
 			}
 			newNode, found := cachedNodes.nodeInfoMap[node.Name]
-			if !found || len(cachedNodes.nodeInfoMap) != 1 {
-				t.Errorf("failed to dump cached nodes:\n got: %v \nexpected: %v", cachedNodes, cache.nodes)
+			if !found || len(cachedNodes.nodeInfoMap) != len(tc.nodes) {
+				t.Errorf("failed to dump cached nodes:\n got: %v \nexpected: %v", cachedNodes.nodeInfoMap, tc.nodes)
 			}
 			expected.Generation = newNode.Generation
-			if diff := cmp.Diff(expected, newNode, cmp.AllowUnexported(framework.NodeInfo{})); diff != "" {
-				t.Errorf("Unexpected clone node info (-want, +got):\n%s", diff)
+			if !reflect.DeepEqual(newNode, expected.Snapshot()) {
```

Comment:
I remember there is a suggestion to use `diff.cmp` in this sig, other reviewers should happy with this change.